src/modeler/AutoRest.Swagger.Tests/SwaggerModelerValidationTests.cs

```diff
@@ -194,6 +194,24 @@ public void InvalidFormatValidation()
         }
 
         [Fact]
+        public void ListOperationsNamingValidation()
+        {
+            var messages = ValidateSwagger(Path.Combine("Swagger", "Validation", "list-operation-naming.json"));
+            messages.AssertOnlyValidationMessage(typeof(ListOperationNamingWarning));
+        }
+
+        [Fact]
+        public void ListByOperationsValidation()
+        {
+            var messages = ValidateSwagger(Path.Combine("Swagger", "Validation", "listby-operations.json"));
+            Assert.Equal(messages.Count(), 3);
```

Comment:
Swap parameters. Expected number always goes first, leading to more readable error messages. Currently:
```
Assert.Equal() Failure
Expected: 4
Actual:   3
```